core/resources/l10n/messages.properties

```diff
@@ -1274,7 +1274,7 @@ TCMotorSelPan.btn.close = Close
 ! PlotDialog
 PlotDialog.CheckBox.Showdatapoints = Show data points
 PlotDialog.lbl.Chart = left click drag to zoom area. mouse wheel to zoom. ctrl-mouse wheel to zoom x axis only. ctrl-left click drag to pan.  right click drag to zoom dynamically.
-
+PlotDialog.btn.saveAsImage = Save As Image
```

Comment:
I know I'm nitpicking, but best to add a whiteline (new line) after this, as to separate the PlotDialog translations and the ComponentTree translations